### DISCUSSION TITLE
change Font Size of Waybar

### DISCUSSION THREAD

**Discussion by root9191** [prasanthrangan/hyprdots#1561] [Q&A]
Hi all,
firstly: a very nice theme for Hyprland i really enjoy it.
But i have an Ultrawide Display and the Icons and Fonts on waybar is way too small. I found one file, where i changed the size and it worked but this was an autogenerated file and so this gets overwritten.
Can someone say where i can set this  up?

**Comment by kRHYME7** [comment]
@root9191

You can either change the ratio of the script. (top part )
or hardcode the font size on the ~/.config/waybar/modules/style.css
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/56f98638-a6c7-4fdc-ac1a-fb6a690077e3)

The correct way to modify this is to  modify `  ~/.config/waybar/config.ctl `   then change the values in the second column, this is the whole waybar scaling this way it won't broke the ratio. 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/a071bd91-9d7b-427d-b018-8f6f4f4a34a1)

**Reply by root9191** [reply]
Thank you i changed the config.ctl and now it looks perfect :)
Just one question: i use Bitwarden as Flatpak and i have dark mode enabled. Every app is dark but Bitwarden has some white accents in the Menu Section - here is a screenshot:

![240603_13h19m37s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/47309435/38b40cf4-8cd3-4733-b57e-a66ab32598bc)

Do you know why is that and how to fix?
Also in your video of this project you have some Color Picker and Apps vor visualizing and playing music.
Can you tell me the names of this app and how i can start them?
